pom.xml

```diff
@@ -0,0 +1,499 @@
+                <goals>
+                  <goal>test-generate</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.graalvm.buildtools</groupId>
+            <artifactId>native-maven-plugin</artifactId>
+            <version>${native.maven.plugin.version}</version>
+            <extensions>true</extensions>
+            <executions>
+              <execution>
+                <id>test-native</id>
+                <phase>test</phase>
+                <goals>
+                  <goal>test</goal>
+                </goals>
+              </execution>
+              <execution>
+                <id>build-native</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>build</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <imageName>${imageName}</imageName>
+              <buildArgs>
+                <buildArg>-H:+ReportExceptionStackTraces</buildArg>
+              </buildArgs>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>native-6.x</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.springframework.boot</groupId>
+              <artifactId>spring-boot-maven-plugin</artifactId>
+              <version>${version.spring.boot}</version>
+              <configuration>
+                <image>
+                  <builder>paketobuildpacks/builder:tiny</builder>
+                  <env>
+                    <BP_NATIVE_IMAGE>true</BP_NATIVE_IMAGE>
+                  </env>
+                </image>
+              </configuration>
+              <executions>
+                <execution>
+                  <id>process-aot</id>
+                  <goals>
+                    <goal>process-aot</goal>
+                  </goals>
+                </execution>
+              </executions>
+            </plugin>
+            <plugin>
+              <groupId>org.graalvm.buildtools</groupId>
+              <artifactId>native-maven-plugin</artifactId>
+              <configuration>
+                <classesDirectory>${project.build.outputDirectory}</classesDirectory>
+                <metadataRepository>
+                  <enabled>true</enabled>
+                </metadataRepository>
+                <requiredVersion>22.3</requiredVersion>
+              </configuration>
+              <executions>
+                <execution>
+                  <id>add-reachability-metadata</id>
+                  <goals>
+                    <goal>add-reachability-metadata</goal>
+                  </goals>
+                </execution>
+              </executions>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
+    </profile>
+    <profile>
+      <id>nativeTest-6.x</id>
```

Comment:
1. Why nativeTest is only necessary for `6.x`?
2. Should we active this profile in GitHub actions?
3. Could you please add more description about spring-native related profiles? These profiles make the pom file too long, may make customer confused.

Reply:
> 1. Why nativeTest is only necessary for `6.x`?
   they are used to simulate the profile `native` and `nativeTest` in spring-boot-starter-parent:3.0.0.
> 2. Should we active this profile in GitHub actions?
   Not necessary.
> 3. Could you please add more description about spring-native related profiles? These profiles make the pom file too long, may make customer confused.
  The user should check each sample's README.md,  we have provided the steps to run the sample and related reference doc.

Reply:
How about writing this information into root README.md? For a customer, it's high possibility that he won't care about the content in `spring-native/xx/README.md`. He will be confused when he sees these contents in the root pom.xml. Put this information in the root README is one solution. Another solution is adding this information as comment in root pom.xml.

Reply:
Updated in root README.md file.